service/dap/server_test.go

```diff
@@ -222,6 +224,14 @@ func TestLaunchStopOnEntry(t *testing.T) {
 
 		// 13 >> disconnect, << disconnect
 		client.DisconnectRequest()
+		oep := client.ExpectOutputEventProcessExited(t, 0)
+		if oep.Seq != 0 || oep.Body.Category != "console" {
```

Comment:
(beyond the scope of this pr) I wonder why check on response's Seq is necessary. DAP isn't specifying the semantic of it, is it an important property that affects the behavior and requires checking? (Afraid this is another case of change-detector tests )

Reply:
This was for consistency with the rest of the test. The idea was to test that we set all the counters correctly. There are a couple of tests that check counters, and the rest do not. Technically we should be setting these (#2245). It did not seem like it mattered for vscode, but looks like other clients might care (https://github.com/mfussenegger/nvim-dap/pull/116). We could define a variable for counter and just increment it with every message instead of hardcoding the numbers.